test/e2e/data/shared/v1alpha4_provider/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+# maps release series of major.minor to cluster-api contract version
+# the contract version may change between minor or major versions, but *not*
+# between patch versions.
+#
+# update this file only when a new major or minor version is released
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
```

Comment:
what is this file doing?

Reply:
it will be used by clusterctl as far as I understand